info-compiler/src/main/java/com/google/sps/webcrawler/WebCrawler.java

```diff
@@ -127,7 +128,7 @@ public void compileNewsArticle(String candidateName, String candidateId, String
     }
   }
 
-  // @TODO [Might adopt in {@code compileNewsArticle}.]
+  // [Might adopt in {@code compileNewsArticle}.]
```

Comment:
Is this not a TODO anymore?

Reply:
My original intention was to indicate that the implementation is not a TODO anymore. It's just that we are currently not using this method. Should I add back the TODO?

Reply:
Oh actually can you just remove the whole commented out method in that case?

Reply:
Gotcha!